README.md

```diff
@@ -22,15 +22,15 @@ appropriate one:
 | Palm             | `>=16.0, <17`     | `main`        | 3.x.x          |
 | Quince           | `>=17.0, <18`     | `main`        | 3.x.x          |
 
-[^1]: For Open edX Maple and Tutor 13, you must run version 13.2.0 or
+[^1]: For Open edX Maple and Tutor 13, you must run version 13.3.0 or
```

Comment:
I think that's something that `bump2version` slipped in. :)

Reply:
oops :) fixed now.